test/integration/targets/ec2_vpc_route_table/tasks/main.yml

```diff
@@ -61,6 +61,33 @@
       <<: *aws_connection_info
     register: nat_gateway
 
+# FIXME:
```

Comment:
Found a bug with check mode while writing tests - cannot create public or private route tables with check mode. This should be fixed before merging.